docs/alloy-forgery/recipe-adaptation.md

```diff
@@ -3,9 +3,9 @@ title: Recipe Adaptation
 project: alloy-forgery
 ---
 
-Since 2.1.0, Alloy Forgery comes with a System to adapt existing eecipes to alloy forging recipes with the only current implementation of the system being used to adapt existing Blast Furnace recipes to work within the forge.
+Since 2.1.0, Alloy Forgery added a system to adapt existing recipes to alloy forging recipes. Currently, the only adapted recipe type for which the system is being using for it's for Blast Furnace recipes to work within the forge.
```

Comment:
"Currently, the only adapted recipe type for which the system is being using **for it's for** Blast Furnace recipes to work within the forge."
I recommend replacing this with something more concise, like:
"Currently, only Blast Furnace recipes are adapted to work with the forge."